gammapy/cube/tests/test_fit.py

```diff
@@ -136,3 +141,53 @@ def test_cube_fit(sky_model, counts, exposure, psf, background, mask, edisp):
 
     assert result.model.spectral_model.parameters.covariance is not None
     assert result.model.spatial_model.parameters.covariance is not None
+
+def test_cube_fit_onebin(sky_model):
+    ebounds = np.logspace(-1.0, 1.0, 2)
+    geom_r = geom(ebounds)
+
+    background_map = background(geom_r)
+    psf_map = psf(geom_r)
+    edisp_map = edisp(geom_r, geom_r)
+    exposure_map = exposure(geom_r)
+    counts_map = counts(sky_model, exposure_map, background_map, psf_map, edisp_map)
+    mask_map = mask(geom_r, sky_model)
+
+    sky_model.parameters["lon_0"].value = 0.5
```

Comment:
Again move this code to the `sky_model` fixture, except for the statement to freeze the index.